cabot/celeryconfig.py

```diff
@@ -27,4 +27,29 @@
     },
 }
 
+CELERY_ROUTES = {
+    'cabot.cabotapp.tasks.run_all_checks': {
+        'queue': 'checks',
+    },
+    'cabot.cabotapp.tasks.run_status_check': {
+        'queue': 'checks'
+    },
+    'cabot.cabotapp.tasks.update_service': {
+        'queue': 'service'
+    },
+    'cabot.cabotapp.tasks.update_instance': {
+        'queue': 'instance'
+    },
+    'cabot.cabotapp.tasks.update_shifts': {
+        'queue': 'batch'
+    },
+    'cabot.cabotapp.tasks.clean_db': {
+        'queue': 'maintenance'
+    },
+}
+
 CELERY_TIMEZONE = 'UTC'
+
+CELERY_RATE_LIMIT = os.environ.get('CELERY_RATE_LIMIT')
+if CELERY_RATE_LIMIT:
+    CELERY_ANNOTATIONS = {"tasks.add": {"rate_limit": CELERY_RATE_LIMIT}}
```

Comment:
`tasks.add` ?? Did you copy-paste the example? :-)

You should mention the actual task name:
```yaml
CELERY_ANNOTATIONS:
      cabot.cabotapp.tasks.clean_db:
        rate_limit: ...
```

You can also apply the settings to all the tasks using a `*`
http://docs.celeryproject.org/en/2.5-archived/configuration.html#celery-annotations